acpdr/swagger-specs/schema-registry.yaml

```diff
@@ -288,7 +470,7 @@ paths:
               "_links":
                 "next": null
         '400':
-          description: Accept header is invalid or missing. Ensure that you are providing a valid accept header for a listing call with no version suppied (e.g. application/vnd.adobe.xed-id+json) before trying again.
+          description: The `Accept` header is invalid or missing. Ensure that you are providing a valid accept header for a listing call with no version supplied (e.g. application/vnd.adobe.xed-id+json) before trying again.
```

Comment:
Should the second mention of accept here also be backticked as `Accept`?

(If it should, there are a few other mentions later on like this.)